frontend/public/components/utils/cog.tsx

```diff
@@ -107,12 +107,14 @@ export class Cog extends DropdownMixin {
       { isDisabled ?
         <Tooltip content="disabled">
           <div ref={this.dropdownElement} className={classNames('co-m-cog', {'co-m-cog--disabled' : isDisabled})} >
-            <span className={classNames('fa', 'fa-cog', 'co-m-cog__icon', {'co-m-cog__icon--disabled' : isDisabled})} aria-hidden="true"></span>
+            <span className={classNames('fa', 'fa-ellipsis-v', 'co-m-cog__icon', {'co-m-cog__icon--disabled' : isDisabled})} aria-hidden="true"></span>
             <span className="sr-only">Actions</span>
           </div>
         </Tooltip> :
-        <div ref={this.dropdownElement} onClick={this.toggle} className={classNames('co-m-cog', {'co-m-cog--disabled' : isDisabled})} >
-          <span className={classNames('fa', 'fa-cog', 'co-m-cog__icon', {'co-m-cog__icon--disabled' : isDisabled})} aria-hidden="true"></span>
+        <div ref={this.dropdownElement} className={classNames('co-m-cog', {'co-m-cog--disabled' : isDisabled})} >
+          <button type="button" aria-label="Actions" aria-haspopup="true" className="btn btn-link co-m-cog__button" onClick={this.toggle}>
+            <span className={classNames('fa', 'fa-ellipsis-v', 'co-m-cog__icon', {'co-m-cog__icon--disabled' : isDisabled})} aria-hidden="true"></span>
```

Comment:
I think we have some unnecessary disabled checks here and above since we have the `isDisabled ?` ternary. You should be able to just set the right `className` as one string without needing `classNames`